azure/index.md

```diff
@@ -62,17 +62,29 @@ layout: default
     tenant=xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
     ```
 
-11. Edit the `user_vars.yml` and fill in required parameters. There are 4 parameters: `node_image`, `node_count`, `node_region` and `node_size`. `node_image` is the image name of Kadena Scalable Permissioned Blockchain. `node_count` refers to the number of nodes you'd like to spin up. `node_region` is the region of the node VMs, and needs to be the same as the Ansible monitor VM. `node_size` can be decided based on the scale of project. For exploration, we recommend Standard B1s. See below as an example:
+11. Edit the `ansible/user_vars.yml` and fill in required parameters. There are 5 parameters:
+    - `node_image`:
+       - `sku`: the Kadena ScalableBFT image SKU Id.
+       - `version`: the Kadena ScalableBFT image SKU version.
+    - `node_count`: the number of nodes you'd like to spin up.
```

Comment:
Add at the end of line 69, The community version of ScalableBFT supports maximum of 4 nodes.